gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -351,6 +357,15 @@ async fn open_channel(
     Ok(Json(json!(())))
 }
 
+#[instrument(skip_all, err, fields(?payload))]
+async fn close_channels_with_peer(
+    Extension(gateway): Extension<Gateway>,
+    Json(payload): Json<CloseChannelsWithPeerPayload>,
+) -> Result<impl IntoResponse, GatewayError> {
+    gateway.handle_close_channels_with_peer_msg(payload).await?;
+    Ok(Json(json!(())))
```

Comment:
Maybe we want to return how many channels were closed?

Reply:
Done.